content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -423,7 +435,7 @@ Webhook authentication is a hook for verifying bearer tokens.
 * `--authentication-token-webhook-config-file` a configuration file describing how to access the remote webhook service.
 * `--authentication-token-webhook-cache-ttl` how long to cache authentication decisions. Defaults to two minutes.
 
-The configuration file uses the [kubeconfig](/docs/concepts/cluster-administration/authenticate-across-clusters-kubeconfig/)
+The configuration file uses the [kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
```

Comment:
Does this page represent the kubeconfig concept:
https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/

Reply:
Yes, this looks better. I was using `/docs/tasks/access-application-cluster/configure-access-multiple-clusters/` because that is where the redirections will lead you to today.